src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -56,6 +56,7 @@ public class DurableTaskExtension :
 
         private EndToEndTraceHelper traceHelper;
         private HttpApiHandler httpApiHandler;
+        private LifeCycleNotificationHelper lifeCycleTraceHelper;

```

Comment:
The variable should be renamed to match the new class name: `lifeCycleNotificationHelper`.

Reply:
Sure. We will change it. Thank you for catching that.